src/server.js

```diff
@@ -13,6 +13,13 @@ const startServer = async (portNumber) => {
   await dbModule.start()
   return new Promise((resolve, reject) => {
     app = express()
+    app.set('view engine','ejs')
+    app.use(express.json())
+    app.get('/',(req,res)=>{
+      const user ={name:'donnell'}
```

Comment:
Who is donnell?

Reply:
Was this `user` object for testing purposes? It doesn't seem like it should be in a commit or code review file.

Reply:
Yea I was thinking whether to just take it out. However in that case we don’t need to use ejs

Reply:
> Who is donnell?

It’s one of my favorite comedians-Donnell Rawlings!